### PR TITLE
Add new post about the new space egress options

### PR DESCRIPTION
**Pull request by apburnes**
To help announce the additional egress options for spaces, outline the reasoning and features of the new space egress types.

_*Wait to merge until egress is in production_

## Changes proposed in this pull request:
- Describes the new egress spaces
- Outlines each space type's ASG rules

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/apb/new-space-egress-post/2021/11/09/controlled-space-egress/)

## Security Considerations
None

### CODE REVIEW

**Comment by mheadd** [comment]
When we say `database service` do we mean any brokered service?

**Comment by apburnes** [comment]
Yes. I'm referring to our brokered services

**Comment by apburnes** [comment]
Updated to use "brokered services"

**Review by mheadd — DISMISSED** [review]
🚀 🇺🇸

File: _posts/2021-11-09-controlled-space-egress.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Here are the following rules associated to the different space ASG types:
+
+- ### `public-egress`
+  - Requests being executed from within the app can successfully be sent to the open internet, our brokered services, and other internal routes you have created in your organization.
+  - Your app can make requests to third party APIs
+  - Your app can connect to our data services
+
+- ### `restricted-egress`
+  - Requests being executed from within the app can only successfully be sent to our brokered services or other internal routes you have created in your organization.
```
Comment (bengerman13) [inline _posts/2021-11-09-controlled-space-egress.md]: should we call out here that S3 is excluded?

Comment (apburnes) [reply _posts/2021-11-09-controlled-space-egress.md]: Sounds good. Also maybe we add a blurb about additional documentation and examples to come.

**Review by bengerman13 — DISMISSED** [review]

File: _posts/2021-11-09-controlled-space-egress.md (hunk 1 above)
Comment (bengerman13) [reply _posts/2021-11-09-controlled-space-egress.md]: I realized you spelled out which services you're talking about above, so I think we're good

File: _posts/2021-11-09-controlled-space-egress.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+date: November 09, 2021
```
Comment (bengerman13) [inline _posts/2021-11-09-controlled-space-egress.md]: I vote update this to today and :shipit: 

```suggestion
date: November 16, 2021
```

**Review by bengerman13 — DISMISSED** [review]

**Review by bengerman13 — APPROVED** [review]

**Pull request merged**